assignAgenttoTicketPart2.js

```diff
@@ -0,0 +1,57 @@
+import assignAgenttoTicket from './assignAgenttoTicket.js';
+import getAgents from './getAgents.js';
+import getTicketsByStatus from './getTicketsByStatus.js';
+import makeZendeskRequest from './makeZendeskRequest.js';
+
+const assignAgenttoTicketPart2 = async (agentLastName) => {
+
+  if (agentLastName == undefined) {
+    agentLastName = '';
+  }
+
+  let agentFirst = 0;
+
+  let agents = await getAgents();
+  console.log(agents);
+
+  if (agentLastName = '') { agentFirst = 0; }
+  else {
+    for (let i = 0; i < agents.length; i++) {
+
+      if (agents[i] == agentLastName) {
+        if (i + 1 == agents.length) agentFirst = 0;
+        else agentFirst = i + 1;
+      }
+    }
+  }
+
+  let agents2 = await makeZendeskRequest('/api/v2//users.json?role[]=agent', 'GET');
```

Comment:
I'd rename agents and agents2 to something that explains the usage e.g 'agentsFromShiftRota' and 'agentsFromZendesk'